first session/assignment/shoppingmall.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>나만의 쇼핑몰</title>
+    <link rel="stylesheet" href="shoppingmall.css">
+</head>
+<body>
+    <header>   
+        <div class="header-icon">🛍️</div>
+        <h1>민준이의 쇼핑몰</h1>
+        <!-- <div class="menu-bar">☰</div> -->
+        <!--<img src="./static/img/menu-bar.png" alt="menu bar">-->
+        <div class="hamburger-menu">
+            <button class="hamburger-icon" onclick="toggleMenu()">
+                ☰
+            </button>
+                   
+        </div>
+    </header>
+        <div class>
+        
+    </div>
+    <nav>
+        <ul>
+            <li><a href="#all">전체</a></li>
+            <li><a href="#category1">운동용품</a></li>
+            <li><a href="#category2">여가생활</a></li>
+            <li><a href="#category3">의류</a></li>
+        </ul>
+    </nav>
+    <main>
+        <div>
+            <h2>전체상품</h2>
+            <section id="all">
+        
+                <div class="product">
+                    <img src="./static/img/자전거.jpg" alt="자전거">
+                    <div><button class="like-button">♥</button></div>
```

Comment:
지금 요기 button을 감싸는 div에 아무런 style도 적용되고 있지 않고, 단순 button 태그 하나를 감싸는 용도로만 사용되고 있어요. 이런 의미없는 div는 사용하지 않는 것이 좋아요!! 밑에 있는 동일한 구조의 태그들도 마찬가지입니다!!
```suggestion
                    <button class="like-button">♥</button>
```